lib/codegen-meta/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cranelift-codegen-meta"
+authors = ["The Cranelift Project Developers"]
+version = "0.15.0"
+description = "DSL for cranelift-codegen code generator library"
+license = "Apache-2.0"
```

Comment:
Also, please add a copy of the LICENSE file under lib/codegen-meta (this is a recent change).